src/proxy.sol

```diff
@@ -64,7 +64,7 @@ contract Proxy is TitleOwned {
         }
     }
 
-    function execute(bytes memory _code, bytes memory _data)
+    function executeWithoutAddress(bytes memory _code, bytes memory _data)
```

Comment:
or `executeBytecode`.

I think it is better to describe what it is instead what it is not.